contracts/OperationsProxy.sol

```diff
@@ -49,80 +61,121 @@ contract OperationsProxy {
 		operations = Operations(_operations);
 	}
 
-	function() public only_owner {
+	function()
+		public
```

Comment:
`relay()` is using `msg.value` I suspect it should be `payable` same as this function.

Reply:
I think both `OperationsProxy` and `SimpleOperations` should have no `payable` functions since they don't do anything with it.